Loop Status Extension/StatusViewController.swift

```diff
@@ -20,33 +20,65 @@ class StatusViewController: UIViewController, NCWidgetProviding {
     @IBOutlet weak var batteryHUD: BatteryLevelHUDView!
     @IBOutlet weak var subtitleLabel: UILabel!
     
+    var context: StatusExtensionContext?
+    var defaults: UserDefaults?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         subtitleLabel.alpha = 0
         subtitleLabel.textColor = UIColor.secondaryLabelColor
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        defaults = UserDefaults(suiteName: Bundle.main.appGroupSuiteName)
+        if let defaults = defaults {
+            defaults.addObserver(
+                self,
+                forKeyPath: defaults.statusExtensionContextObservableKey,
+                options: [.new, .initial],
+                context: nil)
```

Comment:
I wouldn't, foremost because it's convoluting the term "context" for future readers, but secondly because you're looking for a unique pointer value here, and it should feel safer to instead use one containing a simple integer value that never changes.